.bids_conversion/convert.sh

```diff
@@ -0,0 +1,192 @@
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsNoise.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-ReclassifyAsNoise.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/ReclassifyAsSignal.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir_LR_run-1_desc-ReclassifyAsSignal.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_hp2000_clean.dtseries.nii ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-Atlashp2000clean.dtseries.nii || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_MSMAll_hp2000_clean.dtseries.nii ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-AtlasMSMALLhp2000clean.dtseries.nii || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_stats.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-Atlasstats.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_CSF.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-CSF.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_hp2000_clean.nii.gz ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-hp2000clean.nii.gz || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Physio_log.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_physio.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_WM.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-WM.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_CleanedMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-CleanedMGT_atlas.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_HighPassMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-HighPassMGT_atlas.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_NoiseMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-NoiseMGT_atlas.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_OrigMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-OrigMGT_atlas.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_PostMotionMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-PostMotionMGT_atlas.txt || true
+    git mv -fk ${subdir}/MNINonLinear/Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_UnstructNoiseMGT.txt ${newsub}/ses-3T/func/${newsub}_ses-3T_task-rest_dir-LR_run-1_desc-UnstructNoiseMGT_atlas.txt || true
```

Comment:
A bit controversial, feel free to ignore I guess:
FWIW, since renaming anyways (it is sad that they couldn't be modified), might be worth looking into changing extension at least for some of them?  I see there .csv and may be some .tsv (although without headers) files:
```
$> head -n 2 Results/rfMRI_REST1_LR/*.txt             
... stripped since might be sensitive data ;)

==> Results/rfMRI_REST1_LR/ReclassifyAsNoise.txt <==

==> Results/rfMRI_REST1_LR/ReclassifyAsSignal.txt <==

==> Results/rfMRI_REST1_LR/rfMRI_REST1_LR_Atlas_stats.txt <==
TCSName,NumSignal,NumNoise,NumTotal,MEAN,STD,COV,TSNR,OrigVar,HighPassVar,MotionVar,StructNoiseVar,BOLDVar,UnstructNoiseVar,TotalSharedVar,CNR,OrigMGTVar,HighPassMGTVar,PostMotionMGTVar,CleanedMGTVar,OrigMGTbeta,HighPassMGTbeta,PostMotionMGTbeta,CleanedMGTbeta,HighPassVarRatio,MotionVarRatio,StructNoiseVarRatio,BOLDVarRatio,UnstructNoiseVarRatio,OrigMGTVarRatio,HighPassMGTVarRatio,PostMotionMGTVarRatio,CleanedMGTVarRatio,CleanedMGTVarVsBOLDVarRatio,OrigMGTrsq,HighPassMGTrsq,PostMotionMGTrsq,CleanedMGTrsq,UnstructNoiseMGTbeta,UnstructNoiseMGTVar,UnstructNoiseMGTVarRatio,UnstructNoiseMGTrsq,NoiseMGTbeta,NoiseMGTVar,NoiseMGTVarRatio,NoiseMGTrsq,WMbeta,WMVar,WMVarVsBOLDVarRatio,WMCleanedBOLDVar,WMCleanedMGTbeta,WMCleanedMGTVar,WMCleanedMGTVarVsWMCleanedBOLDVarRatio,CSFbeta,CSFVar,CSFVarVsBOLDVarRatio,CSFCleanedBOLDVar,CSFCleanedMGTbeta,CSFCleanedMGTVar,CSFCleanedMGTVarVsCSFCleanedBOLDVarRatio,WMCSFbetaWM,WMCSFbetaCSF,WMCSFVar,WMCSFVarVsBOLDVarRatio,WMCSFCleanedBOLDVar,WMCSFCleanedMGTbeta,WMCSFCleanedMGTVar,WMCSFCleanedMGTVarVsWMCSFCleanedBOLDVarRatio
... and there is more
```
I am not sure if there is a convention for an extension to indicate that file is just a single entry (thus could be .tsv .csv etc but without header).

Why asking, is because in general indication of format or structure in the file is recommended. and that is why BIDS atm has all .tsv's.  (there are some .txt "leaked in" for MEG but that is about it... I put some flame into the fire where it is due).